src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -130,7 +130,13 @@ public enum ErrorCode {
   VECTORIZE_SERVICE_TYPE_UNAVAILABLE("Vectorize service unavailable : "),
   VECTORIZE_USAGE_ERROR("Vectorize search can't be used with other sort clause"),
 
-  VECTORIZECONFIG_CHECK_FAIL("Internal server error: VectorizeConfig check fail");
+  VECTORIZECONFIG_CHECK_FAIL("Internal server error: VectorizeConfig check fail"),
+
+  UNAUTHENTICATED_REQUEST("UNAUTHENTICATED: Invalid token"),
+  INVALID_QUERY("Invalid query"),
+  DRIVER_TIMEOUT("Driver timeout on"),
+  DRIVER_CLOSED_CONNECTION("Driver request connection is closed due to an unrelated event"),
```

Comment:
I don't know what "due to an unrelated event" means, so I doubt a user would either. Are we sure we want to have this as something we report as a separate error? Is this a case where we should do a retry?

Reply:
I think we should re-phrase, or remove "due to an unrelated event". I get what it means -- we did not close it,  but something caused it to close (server-side, proxy) -- but I agree with @jeffreyscarpenter that it probably brings more questions than answers.